modules/plugin/jdbc/jdbc-postgis/src/test/java/org/geotools/data/postgis/PostgisFilterToSQLTest.java

```diff
@@ -200,4 +201,21 @@ public void testEncodeInArrayWithCast() throws Exception {
         String sql = writer.toString().toLowerCase();
         assertEquals("where 5::text=any(testarray)", sql);
     }
+
+    @Test
+    public void TestFunctionLike() throws Exception {
```

Comment:
A method starting with a capital letter?

Reply:
will fix it, apologies